core-google-topic.atom.php

```diff
@@ -0,0 +1,14 @@
+<?php
+require_once __DIR__ . '/google/google-functions.php';
+
+$pub_date = date("c", strtotime( $pub_date ) );
```

Comment:
:no_entry_sign: **Error**: date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead (*WordPress.DateTime.RestrictedFunctions.date_date*).